ci/k8s/run-kuberay-doc-tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo "--- Setup k8s environment"
+SKIP_CREATE_KIND_CLUSTER=1 bash ci/k8s/prep-k8s-environment.sh
+
+echo "--- Install Python dependencies"
+pip install -c python/requirements_compiled.txt pytest nbval bash_kernel
+python -m bash_kernel.install
+pip install "ray[default]==2.41.0"
+
+echo "--- Run doc tests"
+cd doc/source/cluster/kubernetes
+py.test --nbval getting-started/raycluster-quick-start.ipynb --nbval-kernel-name bash --sanitize-with doc_sanitize.cfg
```

Comment:
What's the test plan for kuberay doc test? Are we going to manually add each doc file to this?

Reply:
Currently, yes. I haven't found a good way to identify which files contain doc tests yet. Maybe we can search all `.ipynb` files under the current directory and check whether their kernel is `bash_kernel` to find those that contain doc tests.